### PR TITLE
Lint models/org.go

### PR DESCRIPTION
**Pull request by Bwko**

### CODE REVIEW

File: models/org.go [hunk 1]
```diff
@@ -235,37 +237,37 @@ func DeleteOrganization(org *User) (err error) {
 // OrgUser represents an organization-user relation.
 type OrgUser struct {
 	ID       int64 `xorm:"pk autoincr"`
-	Uid      int64 `xorm:"INDEX UNIQUE(s)"`
+	UID      int64 `xorm:"INDEX UNIQUE(s)"`
```
Comment (lunny) [inline models/org.go:240]: The default `GonicMapper` cannot recognize `UID` as `uid` in database. I think you have to add `UID` to the map in https://github.com/go-gitea/gitea/blob/master/models/models.go#L72

**Review by lunny — CHANGES_REQUESTED** [review]

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by strk** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**